app/package-lock.json

```diff
@@ -1,656 +1,8 @@
 {
   "name": "lisk-desktop",
   "version": "2.3.0-rc.1",
-  "lockfileVersion": 2,
```

Comment:
lockfileVersion 2 is for npm > 7, since we are using node 16 the npm version is 8, so why have you changed it to 1?